src/blocks/author-list/components/inspector/LayoutPanel.js

```
@@ -0,0 +1,146 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	PanelBody,
+	SelectControl,
+	TextControl,
+	RangeControl,
+	__experimentalUnitControl as UnitControl,
```

ESLint

[error] Usage of `__experimentalUnitControl` from `@wordpress/components` is not allowed.
See https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#experimental-and-unstable-apis for details.

(@wordpress/no-unsafe-wp-apis)